src/normLhalf.jl

```diff
@@ -0,0 +1,54 @@
+# Lhalf pseudo-norm (times a constant)
+
+export RootNormLhalf
+
+"""
+**``L_1/2^(1/2)`` pseudo-norm**
+    RootNormLhalf(λ=1)
+Returns the function
+```math
+f(x) = λ\\cdot \\sum |x|\\^{1/2}
+```
+for a nonnegative parameter `λ`.
+"""
+struct RootNormLhalf{R <: Real} <: ProximableFunction
+  lambda::R
+  function RootNormLhalf{R}(lambda::R) where {R <: Real}
+    if lambda < 0
+      error("parameter λ must be nonnegative")
+    else
+      new(lambda)
+    end
+  end
+end
+
+RootNormLhalf(lambda::R=1) where {R <: Real} = RootNormLhalf{R}(lambda)
+
+function (f::RootNormLhalf)(x::AbstractArray{T}) where {T <: Real}
+  return f.lambda * T(sum(sqrt.(abs.(x))))
+end
+
+function prox!(y::AbstractArray{T}, f::RootNormLhalf, x::AbstractArray{T}, gamma::Real=1) where {T <: Real}
+  γλ = 2 * gamma * f.lambda
+  ϕ(z) = acos(γλ / 8 * (abs(z) /3 )^(-3/2))
```

Comment:
```suggestion
  ϕ(z) = acos(2γλ / 8 * (abs(z) / 3 )^(-3/2))
```
Just so it's a bit more explicit.